shell/shell.js

```diff
@@ -16,6 +16,177 @@
 
 const { classes: Cc, interfaces: Ci, results: Cr, utils: Cu } = Components;
 const { Runtime } = Cu.import('resource://qbrt/modules/Runtime.jsm', {});
+const { Services } = Cu.import('resource://gre/modules/Services.jsm', {});
+
+function KeyAccelerators(OS, keyModifierAliases) {
```

Comment:
would these keyboard shortcuts be expected to be used elsewhere? can put them in a common lib/utils file.